### PR TITLE
KNOWN_HEAD_OFFENDER_DOMAINS in config + better test

### PR DESCRIPTION
**Pull request by abulte**
While investigating #44 or similar, this cleans up the hard-coded URLs.

Associated PR for deployment: http://gitlab.incubateur.net/data.gouv.fr/infra/merge_requests/1

### CODE REVIEW

File: croquemort/crawler.py [hunk 1]
```diff
@@ -51,23 +43,24 @@ def check_url(self, url_group_frequency):
             self.storage.store_group(url, group)
             if frequency:
                 self.storage.store_frequency(url, group, frequency)
+        head_timeout = self.config.get('CRAWLER_HEAD_TIMEOUT', HEAD_TIMEOUT)
+        get_timeout = self.config.get('CRAWLER_GET_TIMEOUT', GET_TIMEOUT)
+        no_head_domains = self.config.get('KNOWN_HEAD_OFFENDER_DOMAINS', [])
         try:
```
Comment (davidbgk) [inline croquemort/crawler.py]: Not related to that PR but that `try… except` block is quite heavy.

File: config.yaml [hunk 2]
```diff
@@ -6,6 +6,7 @@ parent_calls_tracked: 10
 REDIS_URI: 'redis://localhost:6379/5'
 CRAWLER_GET_TIMEOUT: 180
 CRAWLER_HEAD_TIMEOUT: 10
+KNOWN_HEAD_OFFENDER_DOMAINS: []
```
Comment (davidbgk) [inline config.yaml]: Not super happy with the setting name, maybe `HEAD_DOMAINS_BLACKLIST` or something in this vein?

File: croquemort/crawler.py (hunk 1 above)
Comment (abulte) [reply croquemort/crawler.py]: Yes :-(

File: config.yaml (hunk 2 above)
Comment (abulte) [reply config.yaml]: Ah ah ;-) Yours is more consensual, I'll give you that :p

**Pull request merged**